cxx-qt-gen/src/syntax/cpp.rs

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+/// A generic C++ header and source pairing
+pub struct CppPair {
```

Comment:
```suggestion
pub struct CppSourceCodePair {
```

Reply:
`CppPair` could be confused with C++'s `std::pair`

Reply:
Done